module.js

```diff
@@ -0,0 +1,136 @@
+const catchA = function () {
+    let name = '';
+    let start = false;
+    let score= 0 ;
+    let level=1 ;
+    let iphones=[];
+    let idCounter=1
+    let timer = 2
+   
+    const randomColor = function () {
+        let ranNum = function () {
+            return Math.floor(Math.random() * 255)
+        }
+        let color = `rgb(${ranNum()},${ranNum()},${ranNum()})`
+        return color
+    }
+
+
+
+    const addIphone = function (num) { 
+        let ranNum = function () {
+            let size = Math.floor(Math.random() * 100)
+
+            if (size<30) {
+               size = 30 
+            } 
+            return size
+        }
+for (let i = 0; i < num; i++) {
+    idCounter++
+    
+    let newIphone = {
```

Comment:
`const`

Reply:
it doesn't matter if the variable is inside a loop, which means it will be reassigned with a new value?

Reply:
You are changing the value inside the object, but you are not changing the fact that `newIphone` is an object. If you were then trying to make `newIphone` a string that would be a problem. Every time it loops it starts over so you should not see an error that says stop assigning a new value to a const `const`